### ISSUE TITLE
`export ... from` does not work as expected with a default export

### ISSUE THREAD

**Issue by gamemaker1** [timocov/dts-bundle-generator#182]
Hi there! Thanks for creating and maintaining this awesome package!

---

### Description

When re-exporting the default export from another file, `dts-bundle-generator` does not `export default` the value.

See a minimal repro [here](https://github.com/gamemaker1/test/tree/dts-bundle-generator).

### Input Code

```ts
// /source/index.ts
// Export away!

// Works if the following is used instead:
// import sayHello from './lib.js'
// export default sayHello

export { default } from './lib.js'
```

```ts
// /source/lib.ts
// Define all library functions

const sayHello = (name: string) => {
	console.log(`'ello there, ${name}!`)
}

export default sayHello
```

### Expected output

```ts
// Generated by dts-bundle-generator v6.2.0

declare const sayHello: (name: string) => void;
export default sayHello; // <== This line should be there

export {};
```

### Actual output

```ts
// Generated by dts-bundle-generator v6.2.0

declare const sayHello: (name: string) => void;

export {};
```

---

I'd be happy to submit a PR to fix this :)

**Comment by timocov** [comment]
> I'd be happy to submit a PR to fix this :)

@gamemaker1 Brilliant, go ahead! 🙂 It seems that it should be handled somewhere [here](https://github.com/timocov/dts-bundle-generator/blob/49a9b071d82053264984918c2c38b98ad2d8848f/src/bundle-generator.ts#L448-L453), but we need to avoid re-export from "outside" modules, i.e. `export { default } from 'package-name'` should be unchanged. Let me know if you want to processed with the fix, otherwise I don't mind to fix it myself.

**Comment by gamemaker1** [comment]
I'll try getting a PR in this week - looks like I need to change some stuff in the ​`updateResultForRootSourceFile​` function in the file you linked. Thanks for the hint!

**Comment by timocov** [comment]
The fix has been deployed as part of [v6.4.0 version](https://github.com/timocov/dts-bundle-generator/releases/tag/v6.4.0). Many thanks @gamemaker1 for all your work on this ❤️
